.github/workflows/tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "*"
```

Comment:
do we want to have tests at all commits or just on PRs? sometimes I just put ABIs in here, tests wouldn't serve much

Reply:
Ideally we run tests for all the commits yeah, this way we keep them updated and we make them run faster 😅 but yeah, not sure if it's worth it running every time here